ada/language/ast.py

```diff
@@ -8192,6 +8192,16 @@ def check_array_slice(typ=T.BaseTypeDecl.entity):
             )
         )
 
+    @langkit_property(public=True, return_type=Bool)
```

Comment:
Just rename the `check_array_slice` into `is_array_slice` and give it a default origin.

Reply:
It's not just about origin, `check_array_slice` takes a parameter. But I agree the name is naze.